packages/stdint/stdint.0.7.0/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+maintainer: ["Markus W. Weissmann <markus.weissmann@in.tum.de>"]
+authors: [
+  "Andre Nathan <andre@digirati.com.br>"
+  "Jeff Shaw <shawjef3@msu.edu>"
+  "Markus W. Weissmann <markus.weissmann@in.tum.de>"
+  "Florian Pichlmeier <florian.pichlmeier@mytum.de>"
+]
+bug-reports: "https://github.com/andrenth/ocaml-stdint/issues"
+homepage: "https://github.com/andrenth/ocaml-stdint"
+doc: "https://andrenth.github.io/ocaml-stdint/"
+license: "MIT"
+dev-repo: "git+https://github.com/andrenth/ocaml-stdint.git"
+synopsis: "Signed and unsigned integer types having specified widths"
+description: """
+The stdint library provides signed and unsigned integer types of various fixed
+widths: 8, 16, 24, 32, 40, 48, 56, 64 and 128 bit.
+
+This interface is similar to Int32 and Int64 from the base library but provides
+more functions and constants like arithmetic and bit-wise operations, constants
+like maximum and minimum values, infix operators conversion to and from every
+other integer type (including int, float and nativeint), parsing from and
+conversion to readable strings (binary, octal, decimal, hexademical), conversion
+to and from buffers in both big endian and little endian byte order."""
+depends: [
+  "ocaml" {>= "4.03"}
+  "qcheck" {test}
```

Comment:
```suggestion
  "qcheck" {with-test}
```

Reply:
Should we kust remover this? Tests are not enabled. 
Also, odoc is missing

Reply:
This should be removed, yes - the tests don’t all pass anyway